doc/source/glossary.rst

```diff
@@ -458,10 +475,6 @@ Glossary
        An alias for a field name in a structured datatype.
 
 
-   type
-       In NumPy, a synonym for :term:`dtype`. For the more general Python
-       meaning, :term:`see here. <python:type>`
-
```

Comment:
This entry has dissapeared on master, so I removed it from the PR as well. OK?

Reply:
This was new content, not meant to be removed.